modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase

```diff
@@ -154,6 +154,342 @@ definition {
+			CommerceCheckout.initializersCheckout(
+				newAddressAddress = "Test Address",
+				newAddressCity = "Test City",
+				newAddressCountry = "United States",
+				newAddressName = "Address Name",
+				newAddressZipCode = "Test Zip Code",
+				productName = "U-Joint",
+				productQuantity = "8",
+				productUnitPrice = "24",
+				shippingMethod = "Standard");
+		}
+	}
+
+	@description = "This is a test for COMMERCE-10503. Minimum quantity can be added to cart when order is lower then stock quantity"
+	@priority = "3"
+	test CanAddedMinimumQuantityWhenOrderIsLowerThenStock{
+		property portal.acceptance = "false";
+
+		task ("Given a buyer user is created") {
+			CommerceEntry.addAccountEntryUser(
+				accountName = "Commerce Account",
+				accountType = "Business",
+				agreeToTermsAndAnswerReminderQuery = "true",
+				createNewAccount = "true",
+				requireReset = "false",
+				userEmailAddress = "buyer@liferay.com",
+				userFirstName = "Buyer",
+				userLastName = "User",
+				userRole = "Buyer",
+				userScreenName = "buyeruser",
+				userSiteMembership = "Minium");
+		}
+
+		task ("And given the warehouse quantity for the product is set to 6") {
+			CommerceProducts.openProductsAdmin();
+
+			CommerceEntry.editWarehousesQuantity(
+				italyQuantity = "2",
+				northeastQuantity = "2",
+				productName = "U-Joint",
+				productSku = "MIN55861",
+				southwestQuantity = "2");
+		}
+
+		task ("And minimum quantity is set to 4 and Allow Back Order is abled") {
+			CommerceProducts.openProductsAdmin();
+			
+			CommerceNavigator.searchEntry(entryName = "U-Joint");
+
+			CommerceNavigator.gotoEntry(entryName = "U-Joint");
+
+			CommerceEntry.gotoMenuTab(menuTab = "Configuration");
+			
+			CommerceProducts.updateProductInventoryConfiguration(
+				allowBackOrders = "true",
+				displayAvailability = "true",
+				displayStockQuantity = "true",
+				minOrderQuantity = "4",
+				publish = "true");
+		}
+
+		task ("And buyer user logs in") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+		}
+
+		task ("And buyer user add minimum quantity to the cart") {
+			FrontStore.addInitializerProductToCart(
+				productName = "U-Joint",
+				productQuantity = "4");
+
+			CommerceCheckout.gotoCheckoutViaMiniCart();
+		}
+		
+		task ("When buyer user can do the checkout with the quantity higher then the stock quantity "){
+			CommerceCheckout.initializersCheckout(
+				newAddressAddress = "Test Address",
+				newAddressCity = "Test City",
+				newAddressCountry = "United States",
+				newAddressName = "Address Name",
+				newAddressZipCode = "Test Zip Code",
+				productName = "U-Joint",
+				productQuantity = "4",
+				productUnitPrice = "24",
+				shippingMethod = "Standard");
+		}
+
+		task("Then product remains available to be add to the cart and buyer user can place a seconde order"){
```

Comment:
task ("Then product remains available to be added to the cart and buyer user can place a second order") {